### ISSUE TITLE
Post page cut off on publish

### ISSUE THREAD

**Issue by y6nH** [Plume-org/Plume#507]
This is a weird one.

I have been writing [this post on fediverse.blog](
https://fediverse.blog/~/YourSixNewHeads/alpha-beta-gamma-trying-out-wendy-carlos's-tunings).

When I saved the draft or published it in its earlier form, the page got cut off part-way through. Not just the post content, the HTML file just ended half way through a word, without any closing tags. This was repeatable and consistent behaviour, in Firefox and Chrome. The full text was still there in the edit view.

The post would publish normally if I removed the banner image, or changed the title. If I edit this post and change the title to the original "Alpha, Beta, Gamma: Trying out Wendy Carlos's tunings", it happens again.

I have reproduced the issue in a draft post with the same content.

- **Plume version:** 0.2.0
- **Operating system:** Windows 10
- **Web Browser:** Firefox 67 and Chrome 73

**Comment by y6nH** [comment]
Adding to this, I returned to the post today. Fediverse.blog has been updated to 0.3.0, and the published version (which was OK when I left it) got cut off again in the process.

**Comment by fdb-hiroshima** [comment]
I don't have the issue on the article you mentioned, but I do have it on [another one](https://fediverse.blog/~/PlumeDev/new-plume-alpha). Maybe it only happen for article wrote by self?

**Comment by y6nH** [comment]
I fixed my article by removing the header image, so it's back to full length. I still have a draft of a broken article, if anyone needs it.

**Comment by fdb-hiroshima** [comment]
I no longer have this issue on the article I linked, do you still have it?

**Comment by y6nH** [comment]
No, it looks as if this is fixed. Fediverse.blog still says it's 0.3.0, so I don't know what changed.

**Comment by serra-allgood** [comment]
Adding to this: With 0.3.0 and viewing in Firefox Developer Edition on macOS Catalina, if my published article has a cover image, it gets cut off towards the bottom; it displays fine in Chrome, however.

**Comment by igalic** [comment]
perhaps this is a bug in Firefox that we should submit to https://github.com/webcompat/web-bugs/ ?

**Comment by serra-allgood** [comment]
I agree that's a good idea, and I think it's worth tracking down what part of the Plume source interacts with Firefox this before doing so

---
Serra C. Allgood
Software Engineer

any historical pronoun set accepted, [xi/xer/xem](https://allgood.link/pronoun_preferences) preferred

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by y6nH** [comment]
I doubt this is a Firefox bug. For one thing, I saw it in Chrome too, and for another, the actual server response is truncated, not just the display of it in the browser. I know that doesn't 100% rule out a browser bug, but I'm not able to reproduce it any more.

**Comment by serra-allgood** [comment]
I am able to consistently reproduce at https://write.allgood.mx/~/ItsAllgoodSerra/practical-cryptography-for-real-people by adding one of the two media images I have uploaded to my instance as an illustration. Adding an illustration causes a caught off in Firefox Developer Edition but not Chrome, both on macOS Catalina.

---
Serra C. Allgood
Software Engineer

any historical pronoun set accepted, [xi/xer/xem](https://allgood.link/pronoun_preferences) preferred

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by fdb-hiroshima** [comment]
I'm pretty sure this come from [rocket_csrf](https://github.com/Plume-org/rocket_csrf) (which we manage), but I've not been able to reproduce reliably to see what is wrong with it.
@serra-allgood the link you provide seems to require some authentication against Yunohost :confused:

**Comment by serra-allgood** [comment]
Ah, sorry, I killed my plume instance not thinking about this thread :(

---
Serra C. Allgood
Software Engineer

any historical pronoun set accepted, [xi/xer/xem](https://allgood.link/pronoun_preferences) preferred

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by y6nH** [comment]
The post text is still available from [Google cache](http://webcache.googleusercontent.com/search?q=cache:https%3A%2F%2Fwrite.allgood.mx%2F~%2FItsAllgoodSerra%2Fpractical-cryptography-for-real-people), if anybody wants to search for clues. I noticed that both Serra's and my articles have a `:` in the title, though that isn't helping me reproduce it.
